src/resources/pathmanager.c

```diff
@@ -4,17 +4,17 @@
 #include <string.h>
 
 #if defined(_WIN32) || defined(WIN32)
-#include <shlobj.h>  //SHCreateDirectoryEx
-#include <shlwapi.h> //PathFileExists
+#include <shlobj.h>  // SHCreateDirectoryEx
+#include <shlwapi.h> // PathFileExistsA
 #else
 #include <sys/stat.h> // mkdir
-#include <unistd.h>
 #endif
 
 #include "resources/pathmanager.h"
 #include "utils/allocator.h"
 #include "utils/c_string_util.h"
 #include "utils/crash.h"
+#include "utils/io.h"
```

Comment:
pathmanager.c could use file_exists to implement pm_in_portable_mode.
might remove the need for some of the above includes such as `<shlwapi.h>`

Reply:
Okay, pm_in_portable_mode() uses file_exists() now.